### ISSUE TITLE
Exception in thread "main" java.lang.UnsatisfiedLinkError: no jniopencv_xfeatures2d in java.library.path

### ISSUE THREAD

**Issue by wei-hai** [bytedeco/javacv#316]
I wrote a small demo to use SIFT in xfeatures2d however when I built the program all other libs were loaded without problem except this one:
```
Caused by: java.lang.UnsatisfiedLinkError: /private/var/folders/py/wjtsvfh11xb18bcyf3l0m_rrvvk7b7/T/javacpp132060842439838/libjniopencv_xfeatures2d.dylib: dlopen(/private/var/folders/py/wjtsvfh11xb18bcyf3l0m_rrvvk7b7/T/javacpp132060842439838/libjniopencv_xfeatures2d.dylib, 1): Library not loaded: @rpath/libopencv_shape.3.0.dylib
  Referenced from: /private/var/folders/py/wjtsvfh11xb18bcyf3l0m_rrvvk7b7/T/javacpp132060842439838/./libopencv_xfeatures2d.3.0.dylib
  Reason: image not found
```
I am using Mac OS X and the version is 10.10.5

**Comment by saudet** [comment]
Ok, could you please try to call `Loader.load(opencv_shape.class)` before using `opencv_xfeatures2d` and see if that works around the problem? Thanks!

**Comment by wei-hai** [comment]
it works, and I also need to call Loader.load(opencv_calib3d.class). Is it an issue building issue?

**Comment by gpostelnicu** [comment]
Hi,

The same issue rises in Scala, where it can be solved by adding:

```
  Loader.load(classOf[opencv_shape])
  Loader.load(classOf[opencv_calib3d])
```

I am happy to send a pull request with this fix if it helps.

**Comment by saudet** [comment]
Thanks @gpostelnicu, but it's not a big deal. I've fixed it in the commit above.

Thanks for reporting @wei-hai!

**Comment by saudet** [comment]
Fixed in version 1.2! Aagain, thanks for reporting.
